test/test_plugin_helper.rb

```diff
@@ -6,3 +6,21 @@
 # Add plugin to FactoryGirl's paths
 FactoryGirl.definition_file_paths << File.join(File.dirname(__FILE__), 'factories')
 FactoryGirl.reload
+
+# Foreman's setup doesn't handle cleaning up for Minitest::Spec
+DatabaseCleaner.strategy = :transaction
+
+class Minitest::Spec
+  class << self
+    alias_method :context, :describe
+  end
+
+  before :each do
+    DatabaseCleaner.start
+    Setting::RemoteExecution.load_defaults
+  end
+
+  after :each do
+    DatabaseCleaner.clean
```

Comment:
why do we need to clean the DB after each test? this will probably slow down the test suite noticeable